### ISSUE TITLE
Is it possible monitoring motion detectors?

### ISSUE THREAD

**Issue by FrankC2000** [MnrGreg/homebridge-paradox-security-system#11]
Can I use this plugin for monitorning motion detectors? I've seen it receives all motion detectors events. It would be great for controlling lights

**Comment by erokoder** [comment]
Hello, Is that possible? Do you try this?

**Comment by saundersrg** [comment]
For monitoring the PIRs you'd use one of the prerequisites for this, https://github.com/Tertiush/ParadoxIP150v2. All events are published to MQTT.

PS - You can't do this if your IP150 has been updated to the 4.10 firmware.

**Comment by erokoder** [comment]
Yes, but I have to create some motion accessories? This plugin can make only Alarm Security System Accessory.

**Comment by jacoscar** [comment]
I was expecting to see the motion sensors as well (was planning to shut down the heating when windows are open); is this pluging just to arm/disarm the alarm then?
